uint32 wrist_goal
float64 wrist_goal_theta